src/multibody/model.hpp

```diff
@@ -503,7 +503,7 @@ namespace se3
     ///
     /// \param[in] model The model structure of the rigid body system.
     ///
-    Data (const Model & model);
+    explicit Data (const Model & model);
```

Comment:
This avoids silly typos being interpreted as type conversion. This should be a compilation error:
```cpp
se3::Data data = model
```